cmd/restic/cmd_backup.go

```diff
@@ -471,7 +473,16 @@ func runBackup(opts BackupOptions, gopts GlobalOptions, args []string) error {
 		Warnf("%s\rwarning for %s: %v\n", ClearLine(), dir, err)
 	}
 
-	_, id, err := arch.Snapshot(context.TODO(), newArchiveProgress(gopts, stat), target, opts.Tags, opts.Hostname, parentSnapshotID)
+	timeStamp := time.Now()
+	if opts.TimeStamp != "" {
+		timeStamp, err = time.Parse(time.RFC3339, opts.TimeStamp)
+		if err != nil {
+			Verbosef("wrong time format\n")
+			return err
```

Comment:
Please return a proper error message using `errors.Fatalf`, like this:
https://github.com/restic/restic/blob/a3f6bf3e5a07949e0a30f1fd05c6d75affe51c08/cmd/restic/cmd_cat.go#L56

You could also include the requested time format in the error (maybe in a second line?) so that users see how it should look like.

What do you think?